alchemist/versions.properties

```diff
@@ -77,9 +93,13 @@ version.junit-jupiter=5.7.0-M1
 version.kotest=4.1.3
 
 version.kotlin=1.3.72
+## # available=1.4-SNAPSHOT
 
 version.net.sf.trove4j..trove4j=3.0.3
 
+version.no.tornado..tornadofx=1.7.20
+##                # available=2.0.0-SNAPSHOT
```

Comment:
uhmm... so are we using the old version? I thought we needed 2.0.0 to work with Java 11+

Reply:
I thought so too, don't remember switching to an older version. Everything works correctly though, and I'm running on Java 11. What should I do?

Reply:
I think we are forcing the version elsewhere...

Reply:
I checked and we are not. What's the output of: `./gradlew alchemist-fxui:dependencyInsight --dependency tornadofx`?

Reply:
This is the output:
```
> Task :alchemist-fxui:dependencyInsight
no.tornado:tornadofx:1.7.20
   variant "compile" [
      org.gradle.status                  = release (not requested)
      org.gradle.usage                   = java-api
      org.gradle.libraryelements         = jar (compatible with: classes)
      org.gradle.category                = library

      Requested attributes not found in the selected variant:
         org.gradle.dependency.bundling     = external
         org.jetbrains.kotlin.platform.type = jvm
         org.gradle.jvm.version             = 11
   ]

no.tornado:tornadofx:{require 1.7.20; reject _} -> 1.7.20
\--- compileClasspath
```

Reply:
This is very strange.